oci/model.py

```diff
@@ -200,6 +200,30 @@ def as_dict(self) -> dict:
         raw = {k:v for k,v in raw.items() if v is not None}
         return raw
 
+    def __hash__(self):
+        annotations = tuple(self.annotations.items()) if self.annotations else ()
```

Comment:
Did you consider normalising (especially sorting) annotations?
If so, why did you decide against it?

Reply:
good point. w/o sorting, I would also have to also enforce strict order-comparison in __eq__